project-fortis-interfaces/src/components/Insights/Maps/HeatMap.js

```diff
@@ -87,6 +91,12 @@ export default class HeatMap extends React.Component {
     this.setState({ placeid: selectedplace.placeid });
   }
 
+  moveMapToBoundingBox(bbox) {
+    const bounds = bbox.length && bbox.length === 4 ? [[bbox[1], bbox[0]], [bbox[3], bbox[2]]] : [];
```

Comment:
This is the same functionality as we're using the constructor https://github.com/CatalystCode/project-fortis/blob/e5415a2a7f168ea074db8a47edcb9c40d2d5e8bb/project-fortis-interfaces/src/components/Insights/Maps/HeatMap.js#L16

Please extract a helper function in this module to remove the duplication and prevent potential future bugs where only one of the locations gets updated.